src/media-groups.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Loops through all supported media groups in master and calls the provided
+ * callback for each group
+ *
+ * @param {Object} master
+ *        The parsed master manifest object
+ * @param {string[]} groups
+ *        The media groups to call the callback for
+ * @param {Function} callback
+ *        Callback to call for each media group
+ */
+export const forEachMediaGroup = (master, groups, callback) => {
```

Comment:
perhaps groups should be optional and include all of them by default?

Reply:
We can do that, but I'm not sure if I've seen a use-case for going through all of them.

Reply:
thats true